test/xcode-specs.js

```diff
@@ -31,6 +31,18 @@ describe('xcode @skip-linux', () => {
     /\d\.\d\.*\d*/.test(version).should.be.true;
   });
 
+  it('should get the path and version again, these values are cached', async function () {
+
```

Comment:
would love to test that retrieving from the cache takes less than 2ms or something like that